contracts/test/_global-hooks.js

```diff
@@ -1,34 +1,56 @@
 const mocha = require("mocha");
 
+const { isForkTest } = require("./helpers");
+
 const _chunkId = Number(process.env.CHUNK_ID);
 const _maxChunks = Number(process.env.MAX_CHUNKS);
```

Comment:
It'd be nice to have these env variables documented in a README

Reply:
Have added them now